src/commands/utility/changeStatus.js

```diff
@@ -0,0 +1,55 @@
+const { SlashCommandBuilder } = require("discord.js");
+
+const MAPPED_STATUS_COMMANDS = {
+  'pr-no-merge': '🚫',
+  'pr-sos': '🆘',
+  'pr-draft': '🚧',
+  'pr-reviewing': '👀',
+  'pr-request-changes': '🔁',
+  'pr-request-review': '❗',
+  'pr-working-in-fixes': '🧑‍🔧',
+  'pr-approved': '✅',
+  'pr-merged': '✅🛫',
+  'pr-merged-need-tasks': '✅🛫📝',
+  'pr-merged-task-created': '✅🛫📋'
+};
+
+const COMMAND_KEYS = Object.keys(MAPPED_STATUS_COMMANDS);
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('changestatus')
+    .setDescription('Change the status of the thread')
+    .addStringOption(option =>
+      option.setName('status')
+        .setDescription('status command')
+        .setRequired(true)
+        .addChoices(
+          COMMAND_KEYS.map(command => ({ name: `/${command}`, value: command }))
```

Comment:
since those aren't actually commands we could remove the slash `/` at the beggining on the `name`, I think would be easier to write it as it

`/change-status status:pr-draft` vs `/change-status status:/pr-draft`

Reply:
in fact you could remove the prefix `pr` and remove the dash `-` on the `name`, to make it more readable. for example:

```js
name: command.replaceAll('pr-', '').replaceAll('-', ' ')
```

so the command would be: `/change-status status:draft` `/change-status status:request changes`